pos_backend_partner/__init__.py

```diff
@@ -0,0 +1 @@
+from . import models
```

Comment:
utf-8 encoding (in all `__init__.py`)
add a line

Reply:
@elicoidal thanks for your review,
Here it's not utf-8, it's only ascii, python 2 support ascii by default without coding mentions
For v11 (python 3), all coding mentions 'll be rejected as default utf-8
Yes it's in the guideline. But there is no added value for the future
Thanks for your comprehension

Reply:
Hi @bealdav I just follow the guidelines (https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#idioms).
When you migrate to v11, you will have to remove the coding line.

We need to stick to the guidelines for consistency over all modules.
I hope you understand

Reply:
done